empty/database/db.js

```diff
@@ -0,0 +1,23 @@
+var knex = require('./knex.js')
+
+var insertRecord = (table, data) => {
+  return knex(table).insert(data).returning('*')
```

Comment:
Since all of these are fairly small functions, I'd omit the braces and return statements:
```js
const insertRecord = (table, data) =>
  knex( table ).insert( data ).returning( * )
```
I don't understand the point of the `then` block in all of these functions either?  You're just immediately returning the parameter, which is the same as just omitting the `then`?